Slingshot.Api/ng/Scripts/app.js

```diff
@@ -393,10 +468,39 @@ angular.module('formApp', ['ngAnimate', 'ui.router'])
     $scope.canMoveToNextStep = function(){
         var isValid = true;
 
-        if (!$scope.formData.tenant || !$scope.formData.subscription || !$scope.formData.params) {
+        if (!$scope.formData.tenant
+            || !$scope.formData.subscription
+            || !$scope.formData.params
+            || (creatingNewRg() && !$scope.formData.existingResourceGroup.name)) {
```

Comment:
Actually this check isn't necessary because it gets taken care of by regex below so I removed it.